api/app/Modules/Library/Http/Controllers/DraftLyricsController.php

```diff
@@ -44,6 +44,12 @@ public function lock(DraftLyrics $draftLyrics): Response
         return response()->noContent();
     }
 
+    public function unlock(DraftLyrics $draftLyrics): Response {
```

Comment:
This should only be allowed for moderators or if the lock is owned by the same user. Otherwise, anyone can call this.

Reply:
Probably using a `$this->authorize('unlock', $draftLyrics);`

Reply:
Good catch, missed this 🤣

Reply:
Done